scripts/partition_manager.py

```diff
@@ -647,11 +647,13 @@ def set_size_addr(entry, size, address):
 
 def set_sub_partition_address_and_size(reqs, sub_partitions):
     for sp_name, sp_value in sub_partitions.items():
-        size = sum([reqs[part]['size'] for part in sp_value['span']])
+        end = max(((reqs[part]['address'] + reqs[part]['size']) for part in sp_value['span']))
+        address = min((reqs[part]['address'] for part in sp_value['span']))
+        size = end - address
+        # size = sum([reqs[part]['size'] for part in sp_value['span']])
```

Comment:
Why is this commented out part included?

Reply:
Thanks, removed